src/pages/taejoon/Detail/Detail.scss

```
@@ -0,0 +1,84 @@
+.container-Detail {
+    margin: 0;
+}
+
+.header {
+    background-color: 	#f4f3ec;
```

```suggestion
    display: flex;
    justify-content: space-between;
    width: 90%;
    margin-left: 5%;
    background-color: 	#f4f3ec;
```
요런 순서대로 나중에 리팩토링 체크리스트 중에서 css속성 순서 파트 부분 확인하시구 반영하시면 더 좋을 거 같아요!